scripts/analyze-and-test-examples.sh

```diff
@@ -34,7 +34,9 @@ echo
 travis_fold start analyzeAndTest.tests.vm
 echo Running VM tests ...
 
-pub run test --reporter expanded --exclude-tags=browser | tee $LOG_FILE
+TEST="pub run test" # --reporter expanded
```

Comment:
What are we disabling here with commenting out `--reporter expanded`, exactly? Might want to just delete it or make a separate comment. This way it looks like a malformed explanation comment.

Reply:
Ah, I had meant to clean that out, thanks. (Running `pub run test` from a script makes `--reporter expanded` the default.)

Done.